src/Umbraco.Infrastructure/PropertyEditors/ImageCropperPropertyValueEditor.cs

```diff
@@ -166,6 +166,11 @@ internal class ImageCropperPropertyValueEditor : DataValueEditor // TODO: core v
                 return null; // clear
             }
 
+            if (editorImageCropperValue is not null && temporaryFileKey.HasValue)
```

Comment:
I guess we should also not call `_temporaryFileService.EnlistDeleteIfScopeCompletes(temporaryFileKey.Value, _scopeProvider)` if file is null